tasks/interactive-bots.md

```diff
@@ -59,14 +59,14 @@ Follow the tutorial below to test run the followup bot on your Zulip instance
   site=<dev-url>
   ```
 
-  Replace the placeholder `<api-key>` in the example with an existing user API
-  key and placeholder `<email>` with the user email. The placeholder `<dev-url>`
-  should point to your development environment URL.
+  In the example replace the placeholder `<api-key>` with an existing API
+  key and placeholder `<email>` with the bot's email. The placeholder 
+  `<dev-url>`should point to your development environment URL.
```

Comment:
I think reversing this sentence makes it confusing - the intention was "replace the <api-key> provided in the example with the API key in your file", but the rephrasing sounds "replace the <api-key> in the example".